symfony/client/src/orangehrmAdminPlugin/pages/education/EditEducation.vue

```diff
@@ -0,0 +1,149 @@
+    <div class="orangehrm-card-container">
+      <oxd-text tag="h6">Edit education</oxd-text>
+
+      <oxd-divider />
+
+      <oxd-form @submitValid="onSave">
+        <oxd-form-row>
+          <oxd-input-field
+            label="Education"
+            v-model="qualification.name"
+            :rules="rules.name"
+          />
+        </oxd-form-row>
+
+        <oxd-divider />
+
+        <oxd-form-actions>
+          <oxd-button
+              type="button"
+              displayType="ghost"
+              label="Cancel"
+              @click="onCancel"
+          />
+          <submit-button />
+        </oxd-form-actions>
+      </oxd-form>
+    </div>
+  </div>
+</template>
+
+<script>
+import {navigate} from '@orangehrm/core/util/helper/navigation';
+import {APIService} from '@/core/util/services/api.service';
+
+export default {
+  props: {
+    educationId: {
+      type: Number,
+      required: true,
+    },
+  },
+  setup() {
+    const http = new APIService(
+        window.appGlobal.baseUrl,
+        'api/v2/admin/educations',
+    );
+    return {
+      http,
+    };
+  },
+  data() {
+    return {
+      isLoading: false,
+      qualification: {
+        id: '',
+        name: '',
+      },
+      rules: {
+        name: [],
+      },
+      errors: [],
+    };
+  },
+  methods: {
+    onSave() {
+    // TODO: Loading
+
+      this.isLoading = true;
+      this.http
+          .update(this.educationId, {
+            name: this.qualification.name,
+          })
+          .then(() => {
+            return this.$toast.success({
+              title: 'Success',
+              message: 'Qualifications updated successfully!',
+            });
+          })
+          .then(() => {
+            this.onCancel();
+            this.isLoading = false;
+          });
+    },
+    onCancel() {
+      navigate('/admin/education');
+    },
+  },
+  created() {
+
+    this.isLoading = true;
+    this.http
+        .get(this.educationId)
+        .then(response => {
+          const {data} = response.data;
+          this.qualification.id = data.id;
+          this.qualification.name = data.name;
+          // Fetch list data for unique test
+          return this.http.getAll();
+        })
+        .then(response => {
+          const {data} = response.data;
+          this.rules.name.push(v => {
+            return (!!v && v.trim() !== '') || 'Required';
+          });
+          this.rules.name.push(v => {
+            return (v && v.length <= 50) || 'Should not exceed 50 characters';
+          });
+          this.rules.name.push(v => {
+            const index = data.findIndex(item => item.name == v);
+            if (index > -1) {
+              const {id} = data[index];
+              return id != this.qualification.id
+                  ? 'Qualification name should be unique'
+                  : true;
+            } else {
+              return true;
+            }
+          });
+          this.isLoading = false;
```

Comment:
`this.isLoading = false` will be called anyway in `finally` block. remove redundant code.